contrib_bots/lib/tictactoe-bot.py

```diff
@@ -0,0 +1,329 @@
+
+from __future__ import absolute_import
+from __future__ import print_function
+import copy
+import random
+from six.moves import range
+
+initial_board = [["_", "_", "_"],
+                 ["_", "_", "_"],
+                 ["_", "_", "_"]]
+
+mode = 'r'  # default, can change for debugging to 'p'
+def output_mode(string_to_print, mode):
+    if mode == "p":
+        print(string_to_print)
+    elif mode == "r":
+        return string_to_print
+
+# -------------------------------------
+class TicTacToeGame(object):
+    smarter = True
```

Comment:
Thanks for the feedback! I didn't/don't intend to make it a setting that could be changed. If the user wants to change smarter because the game is too difficult for them, I might add something in the future so that they can change smarter via a command to the bot. But right now, I think the game has the right degree of difficulty, and there's no need to change this.